app/scripts/controllers/channelsmodal.js

```diff
@@ -339,7 +341,31 @@ app.controller('ChannelsModalCtrl', function ($scope, $modalInstance, $timeout,
 });
 
 // nested controller for the channel basic info tab
-app.controller('channelBasicInfoCtrl', function ($scope, $timeout, Api, Notify, Alerting) {
+app.controller('channelBasicInfoCtrl', function ($scope, $timeout, $interval, Api, Notify, Alerting) {
+
+
+  var updateTime = function(timezone) {
+    $scope.aboutInfo.serverTime = getTimeForTimezone(timezone);
+    $scope.aboutInfo.serverTimezoneOffset = getTimezoneOffset(timezone);
+  };
+
+  var success = function(result) {
+    $scope.aboutInfo = result;
+    updateTime(result.serverTimezone);
+    $scope.clock = $interval(function(){
+      updateTime(result.serverTimezone);
+    }, 1000);
+  };
+
+  var error = function(err) {
+    Alerting.AlertAddServerMsg(err.status);
+  };
+
+  Api.About.get(success, error);
+
+
```

Comment:
Do we not need to cancel the interval on destroy like you did in about controller?  Is it different for modal?